Changelog

```diff
@@ -1,3 +1,60 @@
+Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.6.0
+    - FEATURE: Add (experimental) I2P support (glassez) cdded6cef7710fb25093ea142edea3d3b7479447
+    - FEATURE: Provide UI Theme editor (glassez) 77aa85fbd3af12b319f8f0b8372f4ba186d10c80
+    - FEATURE: Various UI theming improvements (glassez)
+    - FEATURE: Implement torrent tags editing dialog (glassez) d40be79c6933fd911644b0b0c833893ea480dc90
+    - FEATURE: Implement "Add torrent params" editing widget (glassez) 0cc29f18510b084b131fd0b02963a7aff909de01
+    - FEATURE: Revamp "Watched folder options" and "Automated RSS downloader" dialog (glassez) 93a1e58554fe9beff657b1302cc358722e8b0a1b 905f1416572913cf0bb0d143b7da381cd544fad8
+    - FEATURE: Allow to use another icons in dark mode (glassez) 72ac92ec682b96924bdec606bc4383430f3f7067
```

Comment:
I think this is included in "Provide UI Theme editor". Also it's incomplete because icons can be changed in light mode too.

Reply:
It means that it's possible use different icon sets for light and dark modes. It's independent from theme editing. Theoretically qBittorrent can now have alternative icons for dark mode if someone provide them.